src/main/groovy/net/anshulverma/gradle/estilo/checkstyle/checks/CheckType.groovy

```diff
@@ -18,14 +18,24 @@ package net.anshulverma.gradle.estilo.checkstyle.checks
 /**
  * @author Anshul Verma (anshul.verma86@gmail.com)
  */
-public enum CheckType {
+public class CheckType {
```

Comment:
So it seemed that using an `enum` was going against the nature of the `CheckType`s being able to be defined during the runtime of the gradle script. I tried to get the `enum`-like behavior by representing the previously-enumerated values as public static fields.